### PR TITLE
Controller testsBlog app - controllers specs

### PR DESCRIPTION
**Pull request by shafiuyushawu**
Hello Reviewer
My name is @shafiuyushawu and in this milestone below requirement was accomplished.

- Add RSpec as a gem to your project.
  - Create a Request spec file for all your controllers.
  - Make sure that for each action you check:
  - If response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('List of post by a user')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success and view' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Posts of specific user')
```
Comment (rloterh) [inline spec/requests/posts_spec.rb:16]: - [x] A request post spec is failing. Let's ensure to fix it. I'll also urge you to fix the additional failing view spec as seen from the attached screenshot.

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Shafiu, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!
🏁 🚦 🏃 🌴 🏫 🌳 🏘️ 

## Highlights 👍  
- Good job, controller specs are almost in shape!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Let's improve the look of the ReadMe by removing the circled aspect, as seen below:

image

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (shafiuyushawu) [reply spec/requests/posts_spec.rb:16]: @rloterh Thanks for the insights, it has been resolved.

Comment (peterrobert) [reply spec/requests/posts_spec.rb:16]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @shafiuyushawu, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shafiuyushawu** [comment]
@peterrobert Thank you for the approval 👍

**Pull request merged**